python/src/com/jetbrains/python/psi/stubs/PyClassAttributesIndex.java

```diff
@@ -30,6 +34,36 @@
     return StubIndex.getElements(KEY, name, project, GlobalSearchScope.allScope(project), PyClass.class);
   }
 
+  public static Collection<PyTargetExpression> findClassAtrributes(@NotNull String name,
```

Comment:
There is no need in distinguishing class and instance attributes since for all usages we are interested in both. It won't be hard to split getting class / instance attributes into two methods if we ever need it, but for now I would prefer a simpler version with just one method.